File: assets/rest/samples/python/TransactionsAPI.getUnconfirmedTx.get.py
from risesdk import APIError, Client

api = Client('https://wallet.rise.vision/api/')
try:
    utx = api.transactions.get_unconfirmed_transaction('6920969059388666996')
    if utx is not None:
        print(utx.tx.fee) # 10000000
except APIError as err:
    print('API error:', err)

File: assets/rest/samples/elixir/TransactionsAPI.getUnconfirmedTx.get.exs
import RiseElixir

try do
  result = RiseElixir.Transactions.get_unconfirmed_by_id(%{id: "6920969059388666996"})

  if result["error"] do
    # "Transaction not found in this queue 6920969059388666996"
    IO.puts(result["error"])
  else
    IO.puts(Enum.at(result["transactions"], 0))
  end
catch
  err -> "API error #{err}"
end
